tee-worker/scripts/litentry/release/deploy.sh

```diff
@@ -0,0 +1,640 @@
+
+      # Move the service to systemd
+      cp -r "worker${i}.service" ~/.config/systemd/user
+      systemctl --user daemon-reload
+      echo "Starting worker service"
+      cd ~/.config/systemd/user/
+      systemctl --user start "worker${i}".service
+    done
+}
+
+function stop_old_worker(){
+  TIMEOUT=300  # Timeout in seconds
+  SECONDS=0
+  while (( SECONDS < TIMEOUT )); do
+      LOG_FILE="log/worker0.log"
+        if grep -q "Enclave did not produce sidechain blocks" "$LOG_FILE"; then
+            echo "Enclave has stoppped producing blocks, Stopping it now"
+            worker_count=$(echo "$CONFIG" | jq '.workers | length')
+
+            for ((i = 0; i < worker_count; i++)); do
+              systemctl --user stop "worker${i}".service
+            done
+            echo "Sleeping for 120 seconds, So that old worker can be stopped gracefully.."
+            sleep 120
+        fi
+      sleep 10
+  done
+}
+
+function migrate_worker(){
+  # Copy integritee-service binary and enclave_signed.so to ./tmp/w0
+  cp ./bin/integritee-service ./tmp/w0
+  cp ./bin/enclave.signed.so  ./tmp/w0
+  cd ./tmp/w0 || exit
+
+  echo "Old MRENCLAVE VALUE: $OLD_MRENCLAVE"
+  echo "New MRENCLAVE VALUE: $NEW_MRENCLAVE"
+  # Run the migration command
+  ./integritee-service migrate-shard --old-shard $OLD_MRENCLAVE --new-shard $NEW_MRENCLAVE
+
+  # Navigate to ./tmp/w0/shards
+  cd shards || exit
+
+  # Find the two files and delete the older one
+  files=(*)
+  if [[ ${#files[@]} -eq 2 ]]; then
+      file1="${files[0]}"
+      file2="${files[1]}"
+      if [[ $file1 -ot $file2 ]]; then
+          echo "Deleting the older shard folder: $file1"
+          rm -r "$file1"
+      else
+          echo "Deleting the older shard folder: $file2"
+          rm -r "$file2"
+      fi
+  fi
+
+  echo "Migration of shards completed"
+}
+
+
+
+function latest_sidechain_sync_block(){
+  # Fetch Latest Block Produced
+  line=$(grep '\[.*\]$' $ROOTDIR/tee-worker/log/worker0.log | tail -n 1 2>&1)
+  number=$(echo "$line" | sed -E 's/.*\[([0-9]+)\]$/\1/')
+  current_sidechain_end_block=$((number + 50))
+  echo "The next enclave is scheduled to start producing blocks after: $current_sidechain_end_block blocks"
+  export SCHEDULE_UPDATE_BLOCK="$current_sidechain_end_block"
+}
+
+function latest_parentchain_sync_block(){
+  # JSON-RPC request payload
+  request='{"jsonrpc":"2.0","id":1,"method":"chain_getHeader","params":[]}'
+
+  # Make the JSON-RPC request and retrieve the latest finalized block
+  response=$(curl -s -H "Content-Type: application/json" -d "$request" http://$PARACHAIN_HOST:$PARACHAIN_PORT)
+  hex_number=$(echo "$response" | grep -oP '(?<="number":")[^"]+')
+  dec_number=$(printf "%d" "$hex_number")
+
+
+  # Store the latest finalized block numbe  r in an environment variable
+  export LATEST_FINALIZED_BLOCK=${dec_number}
+
+  echo "Latest finalized block number: $LATEST_FINALIZED_BLOCK"
+}
+
+function build_parachain(){
+  cd $ROOTDIR
+  make build-node
```

Comment:
`build-node` is with `tee-dev` feature active - if on production you probably need to compile it without this feature flag.

Additionally, I believe copy from the docker image `litentry/litentry-parachain:tee-prod` would be easier, maybe you want to add an extra flag to control if you want to build the parachain binary from source or copy from docker image

Reply:
So building is an optional flag with `--build`, If we want to use separate build process, we can pre-build. I'll look into getting the binary

Reply:
Done, 
So added a flag `--copy-from-docker` so user can enter their own docker image to pull from else it will default to `litentry/litentry-parachain:tee-prod`

Reply:
Included `Production` build that builds without the `tee-dev` feature